modules/hcp-virt-access.adoc

```diff
@@ -10,7 +10,7 @@ You can access the hosted cluster by either getting the `kubeconfig` file and `k
 
 .Prerequisites
 
-To access the hosted cluster by getting the `kubeconfig` file and credentials directly from resources, you need to be familiar with the access secrets for hosted control plane clusters. The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and the _hosted control plane_ namespace is where the hosted control plane runs.
+To access the hosted cluster by getting the `kubeconfig` file and credentials directly from resources, you need to be familiar with the access secrets for hosted clusters. The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and the _hosted control plane_ namespace is where the hosted control plane runs.
```

Comment:
"The secrets are stored in the hosted cluster (hosting) namespace. The _hosted cluster (hosting)_ namespace contains hosted cluster resources, and "

The hosted cluster (hosting) namespace contains the hosted cluster resources, including the access secrets. The _hosted control plane_ namespace is where the hosted control plane runs.